woocommerce/woocommerce-paypal-payments
# PUI conflict with Germanized plugin and taxes

Following the setup of the Germanized plugin, users will see an checkout error in certain conditions when product and shipping have 2 decimals in the amount while attempting the purchase with PUI.

### Steps to reproduce

- Install WooCommerce
- Set store country and currency to Germany and Euro
- Install latest PayPal Payments for WooCommerce plugin
- Onboard with PUI feature 
- Enable PUI feature
- Install Germanized plugin
- Finish the setup
- Enable tax
- Enable shipping 
- Add shipping zone with 10.10 euro flat rate 
- Create a taxable simple product with 549.12 euro price
- Navigate to checkout page
- Select PUI gateway
- Input billing details and birth date
- Place the order

Observe, error is thrown:
`REQUIRED_PARAMETER_FOR_PAYMENT_SOURCE /purchase_units/0/items The parameter is required for provided payment source.`
